src/StripedBackupSet.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace AgDatabaseMove
+{
+
+  public class StripedBackupSet
+  {
+    public IEnumerable<BackupMetadata> StripedBackups { get; private set; }
+
+    public StripedBackupSet(IEnumerable<BackupMetadata> backups, BackupMetadata backupToMatch)
+    {
+      var stripes = backups.Where(backup => AreTwoBackupsStriped(backup, backupToMatch));
+      StripedBackups = new List<BackupMetadata>(stripes);
+    }
```

Comment:
This should be a private constructor; ie. nothing should be calling this outside of this class itself